src/renderer/components/+workspaces/workspaces.tsx

```diff
@@ -111,7 +115,7 @@ export class Workspaces extends React.Component {
           <Trans>Workspaces</Trans>
         </h2>
         <div className="items flex column gaps">
-          {this.workspaces.map(({ id: workspaceId, name, description, ownerRef }) => {
+          {this.workspaces.map(({ id: workspaceId, name, description, ownerRef, enabled }) => {
```

Comment:
`!enabled` should be hidden.

Reply:
@jakolehm What is the reason for this? What we should do with significant delay before new workspace show up in the list?

Reply:
Store might have stale workspaces that extensions have created (`enabled === false`), we don't want to show them in UI.

Reply:
Fixed
![add workspace](https://user-images.githubusercontent.com/9607060/99390474-be4adb80-28e9-11eb-8801-5798ca7d7ce0.gif)